# Global parameters

# Perturbation Amount Parameters
# FOR TENT BUILDING
delta_x: 0.1 # meters
delta_y: 0.1 # meters
delta_z: 0.1 # meters
delta_th_x: 0.1745329 # 0.0174533 # 0.0349066 # 0.1745329 # rad
delta_th_y: 0.1745329 # 0.0174533 # 0.0349066 # 0.1745329 # rad
delta_th_z: 0.1745329 # 0.0174533 # 0.0349066 # 0.1745329 # rad

# # FOR MINGRUI YU EXPERIMENTS
# delta_x: 0.01 # 0.1 # meters
# delta_y: 0.01 # 0.1 # meters
# delta_z: 0.01 # 0.1 # meters
# delta_th_x: 0.0349066 # 0.0174533 # 0.0349066 # 0.1745329 # rad
# delta_th_y: 0.0349066 # 0.0174533 # 0.0349066 # 0.1745329 # rad
# delta_th_z: 0.0349066 # 0.0174533 # 0.0349066 # 0.1745329 # rad

# 0.034906 radians = 2 degrees
# 0.0174533 radians = 1 degree
# 0.1745329 radians = 10 degrees

# pub_rate_odom: 500


